modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/format_storage.es.js

```diff
@@ -0,0 +1,57 @@
+const DEFAULT_OPTIONS = {
+	addSpaceBeforeSuffix: false,
+	decimalSeparator: '',
+	denominator: 1024.0,
```

Comment:
Denominator should not be a settable option. It is not likely that kilobyte definition will change any time soon.